docs/docs/sourcing-from-agilitycms.md

```diff
@@ -46,52 +50,274 @@ Install the dependencies:
 
-All of those pages and content are then made available in GraphQL to the React Components you will write to render those pages.
+For example, if a Page Template has a reference name of MainTemplate in the CMS, then while the page is being rendered by the gatsby-source-agilitycms plugin, it will look for ./src/components/agility-pageTemplates/MainTemplate.jsx in the ./src/components/agility-pageTemplates directory.
 
-Check out the component called "Jumbotron". This is an example of how to display a styled heading and sub-heading with content that comes from Agility CMS. Here is the Module that provides this content being edited in the Agility CMS Content Manager:
+**MainTemplate.jsx:**
 
-![Agility CMS - Example Module - Jumbotron](./images/agilitycms-jumbotron.png)
+```shell
+import React from "react"
+import ContentZone from "../../agility/components/ContentZone"
+import { getModule } from "../../components/agility-pageModules"
+
+const MainTemplate = props => {
+  return (
+    <div id="main-template">
+      <ContentZone name="MainContentZone" {...props} getModule={getModule} />
+    </div>
+  )
+}
+export default MainTemplate
+```
+
+[Learn more about How Agility Page Templates Work in Gatsby](https://help.agilitycms.com/hc/en-us/articles/360043038271)
+
+### Resolving Linked Content
 
-And here is the code used to render it. Notice that the `title` and `subTitle` fields are available as properties of the `item.fields` object.
+While you are sourcing and querying content in gatsby, you are likely to come across a scenario where you need to retrieve a content item and its related content. In Agility CMS, Linked Content fields are used to related content to one another in various ways.
 
-```jsx:title=src/modules/Jumbotron.js
-import React, { Component } from "react"
-import { graphql, StaticQuery } from "gatsby"
+When querying a Post, for example, you may want to also retrieve the details for the Category. On your Post GraphQL node, you may notice a category property, however, it will only contain a contentID reference, not the entire node representing the Category. You'll need to resolve this Linked Content when you need it.
 
-import "./Jumbotron.css"
+In the [agilitycms-gatsby-starter](https://github.com/agility/agilitycms-gatsby-starter), we resolve Linked Content by using [Gatsby Resolvers](https://www.gatsbyjs.org/blog/2019-03-04-new-schema-customization/).
 
-export default class Jumbotron extends Component {
-  render() {
-    return (
-      <section className="jumbotron">
-        <h1>{this.props.item.fields.title}</h1>
-        <h2>{this.props.item.fields.subTitle}</h2>
-      </section>
-    )
+Resolvers are added to your gatsby-node.js in your site, and they allow you to add a new field to your content node which will handle resolving your Linked Content reference. This means you are telling GraphQL, when you query a specific property on a node, it will actually run a function to go and get your Linked Content and return it.
```

Comment:
```suggestion
Resolvers are added to your `gatsby-node.js` in your site, and they allow you to add a new field to your content node which will handle resolving your Linked Content reference. This means you are telling GraphQL, when you query a specific property on a node, it will actually run a function to go and get your Linked Content and return it.
```